hitch-2011/hitch-fe
# Feature/form

## Is this a fix or a feature?
Feature 

## What is the change?
Set up a reusable Form component for user inputs allowing a user to add name, email, username, make, model, year, origin, and destination. Added error handling so a user cannot advance or enter the form until all inputs are completed. Connect the next button so that it advances to the next set of inputs. 

## What does it fix?
n/a

## Where should the reviewer start?
Review App.tsx and Form.tsx to see form changes 

## How should this be tested?
Open app with npm start and advance through form inputs. Check error handling by trying to click the next button without filling out all forms. 

## Relevant ScreenShots
